circle.yml

```diff
@@ -19,12 +19,10 @@ machine:
   java:
     version: openjdk8
   post:
-    # Manually install very recent version of cassandra; unless we create a user we have to run it as root
-    # uncomment below when datastax has version 3.9 available. at time of commit, latest was 3.7
-    #- curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
-    - curl -SL http://cassci.datastax.com/job/trunk/lastSuccessfulBuild/artifact/build/apache-cassandra-3.10-SNAPSHOT-bin.tar.gz | tar xz
-    - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
-    - sudo ./apache-cassandra-*/bin/cassandra -R
+    # Manually install very recent version of cassandra
+    - curl -SL http://downloads.datastax.com/community/dsc-cassandra-3.0.8-bin.tar.gz | tar xz
+    - sed -i -e 's/INFO/WARN/g' dsc-cassandra-*/conf/logback.xml
+    - sudo ./dsc-cassandra-*/bin/cassandra
```

Comment:
don't install the datastax download.
please continue with the latest 3.9-SNAPSHOT until it is released (very soon).

the correct link is actually http://cassci.datastax.com/job/cassandra-3.9/lastSuccessfulBuild/artifact/build/apache-cassandra-3.9-SNAPSHOT-bin.tar.gz

I have no objection to this PR waiting until 3.9 is on the official apache.org downloads page before being merged to master.

As I've stated elsewhere I do object to reverting back to 3.0.8.